src/Symfony/Component/PropertyInfo/Extractors/DoctrineExtractor.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ * (c) Kévin Dunglas <dunglas@gmail.com>
```

Comment:
I'm no license expert, but I think it would be better to be consistent with everything. We can talk about that on the phone.